includes/class-llms-case.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * LLMS_Case definition
+ *
+ * @package LifterLMS/Classes
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Methods to help with changing the case of strings.
+ *
+ * @since [version]
+ */
+class LLMS_Case {
+
+	/**
+	 * Change all characters in the string to lowercase.
+	 *
+	 * @var int
+	 */
+	const LOWER = 0;
+
+	/**
+	 * Do not change the case of the string.
+	 *
+	 * @var int
+	 */
+	const NO_CHANGE = 1;
+
+	/**
+	 * Change all characters in the string to uppercase.
+	 *
+	 * @var int
+	 */
+	const UPPER = 2;
+
+	/**
+	 * Change the first character in the string to uppercase.
+	 *
+	 * @var int
+	 */
+	const UPPER_FIRST = 3;
+
+	/**
+	 * Change the first character of each word in the string to uppercase.
+	 *
+	 * @var int
+	 */
+	const UPPER_WORDS = 4;
+
+	/**
+	 * Changes the case of a string.
+	 *
+	 * @since [version]
+	 *
+	 * @param string $string The string to change the case of.
+	 * @param string $case   One of the CASE_ constants from {@see LLMS_Case}.
+	 * @return string
+	 */
+	public static function change( $string, $case ) {
```

Comment:
This looks great, only change I'd suggest would be to split the conditions into individual (private) functions: 1 for multibyte function usage and one for non mb usage. This will make testing easier while also slimming down the method's length.